modules/apps/mobile-device-rules/mobile-device-rules-test/src/testIntegration/java/com/liferay/mobile/device/rules/service/test/MDRRuleGroupLocalServiceTest.java

```diff
@@ -65,6 +75,111 @@ public void setUp() throws Exception {
 		_group = GroupTestUtil.addGroup();
 	}
 
+	@Test
+	public void testCopyRuleGroup() throws Exception {
```

Comment:
@yuhai Could you confirm that the integration test can expose the issue before fixing? That means the integration test should fail if the local service doesn't copy the permissions.

Reply:
@dantewang, testCopyRuleGroup method can't expose the issue, it tested copy default permissions. testCopyRuleGroupWithoutOwnerViewPermission method can expose the issue before fixing.

Reply:
@yuhai Could you add the integration test to expose the issue first, and then fix it? Also, I think it's OK to just test copying the MDRRuleGroup with modified permissions -- compare every permission to make sure all of them are equal (copied).

Reply:
@dantewang, ok, I will do this.